src/components/settings/services/EditServiceForm.js

```diff
@@ -398,6 +406,12 @@ class EditServiceForm extends Component {
                   field={form.$('customIcon')}
                   textDelete={intl.formatMessage(messages.iconDelete)}
                   textUpload={intl.formatMessage(messages.iconUpload)}
+                  maxSize={2_097_152}
```

Comment:
Does it make sense to hardcode this number or use `Number.POSITIVE_INFINITY`?

Reply:
The function by default uses it as infinity, so I thought we should parse the same. Tho, maybe we can change that if it's unwanted.

The limit is hardcoded there because it's component specific (as we might have different sizes for different components) if we add it somewhere else.